src/Process/3dTilesProcessing.js

```diff
@@ -33,14 +33,14 @@ function subdivideNode(context, layer, node) {
                     node.add(tile);
                     tile.updateMatrixWorld();
                     if (node.additiveRefinement) {
```

Comment:
For my personal culture, why do we `notifyChange` only if node.additiveRefinement is true?